package.json

```diff
@@ -38,28 +37,35 @@
   },
   "homepage": "https://github.com/Significa/react-snuggle#readme",
   "devDependencies": {
-    "@significa/eslint-config-significa": "^1.0.0",
-    "@storybook/addon-knobs": "^3.4.10",
-    "@storybook/addon-storysource": "^3.4.10",
-    "@storybook/addons": "^3.4.10",
-    "@storybook/react": "^3.4.10",
-    "@storybook/storybook-deployer": "^2.3.0",
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^9.0.0",
-    "babel-plugin-external-helpers": "^6.22.0",
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/preset-react": "^7.0.0",
+    "@storybook/addon-knobs": "^4.1.11",
+    "@storybook/addon-storysource": "^4.1.11",
+    "@storybook/addon-viewport": "^4.1.11",
+    "@storybook/react": "^4.1.11",
+    "@types/react": "^16.7.22",
+    "awesome-typescript-loader": "^5.2.1",
```

Comment:
You're not using this. Also, you might need @types/storybook__react. (and more for each storybook plugin)

From a first glance I think there are a lot of dependencies here that are not needed.
Do we need css-loader, style-loader, styled-components?

There's also babel-core and @babel/core (and presets)

Reply:
The babel plugins are necessary for the rollup and storybook, and the `styled-components` is used in the storybook in an example.